google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -103,10 +110,12 @@ public static StorageException toStorageException(
     }
     switch (error.getCode()) {
       case STREAM_FINALIZED:
-        return new StreamFinalizedException(error.getEntity(), error.getErrorMessage(), exception);
+        return new StreamFinalizedException(
+            Status.INVALID_ARGUMENT, null, error.getEntity(), error.getErrorMessage(), exception);
```

Comment:
The status should be derived from the actual rpcStatus, because we are not sure what kind of status code it actually would return (can be preconditioned_failure)

Reply:
Updated